components/prism-keyman.js

```diff
@@ -0,0 +1,14 @@
+Prism.languages.keyman = {
+	'comment': /\b[cC]\s.*?(\r?\n|$)/,
```

Comment:
You can simplify the regexp by using the `i` flag.
Also, I think you can remove the last part `?(\r?\n|$)` since the dot `.` does not match line feeds. (Unless you intentionally wanted to consume the line feed?)